### PR TITLE
Update lazer medal support (`2024.312.0`)

### PR DESCRIPTION
**Pull request by LeaPhant**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

**Comment by chromb** [comment]
it doesn't quite seem correct to change this just yet, for as far as i know there are some hush-hush medals that aren't implemented yet. Probably best to hold off until parity with stable.

**Comment by marcostudios** [comment]
> it doesn't quite seem correct to change this just yet, for as far as i know there are some hush-hush medals that aren't implemented yet. Probably best to hold off until parity with stable.

How about "Partial"?

**Comment by Hiviexd** [comment]
should definitely be partial at the moment.

**Comment by peppy** [comment]
There's around 15 left.

File: wiki/Help_centre/Upgrading_to_lazer/en.md [hunk 1]
```diff
@@ -342,6 +342,7 @@ You're likely thinking of another game.
 [^countdown-timers-lazer]: Set a countdown from the game UI to automatically start the match.
 [^queue-modes]: Turn on to allow anyone in a lobby to queue new beatmaps, a.k.a. "host rotate".
 [^difficulty-adjust]: Change CS/AR/OD/HP of a beatmap directly from song select via the Difficulty Adjust mod.
+[^medals-lazer]: A small amount of medals is not yet obtainable.
```
Comment (cl8n) [inline wiki/Help_centre/Upgrading_to_lazer/en.md:345]:
```suggestion
[^medals-lazer]: Some [Hush-Hush medals](/wiki/Medals#hush-hush) are not yet obtainable.
```

Comment (cl8n) [reply wiki/Help_centre/Upgrading_to_lazer/en.md:345]: also need to outdate all of the translations because of this note

**Review by cl8n — APPROVED** [review]

**Review by Walavouchey — APPROVED** [review]

**Review by peppy — APPROVED** [review]

**Pull request merged**